service/k8sclient/k8sclient.go

```diff
@@ -2242,3 +2152,300 @@ func (c *K8sClient) makeReq112Plus(params *PSMDBParams, extra extraCRParams) *ps
 
 	return req
 }
+
+func (c *K8sClient) createPSMDBSpec(operator *goversion.Version, params *PSMDBParams, extra *extraCRParams) (*psmdb.PerconaServerMongoDB, error) {
+	spec := new(psmdb.PerconaServerMongoDB)
+	bytes, err := ioutil.ReadFile(psmdbCRFile)
+	if err == nil {
+		err = c.unmarshalTemplate(bytes, spec)
+		if err != nil {
+			return nil, err
```

Comment:
do we need hard exit here ? Maybe just message that template isn't good and continue with default?
maybe better to get Max feedback - what he will prefer to test those templates as well...